### PR TITLE
Update auth settings to default to `IsAuthenticated`

### PR DESCRIPTION
**Pull request by robhudson**

### CODE REVIEW

File: backend/api/views.py [hunk 1]
```diff
@@ -357,6 +358,7 @@ def metric_by_id(request, exp_id, metric_id):
 
 
 @api_view(['POST'])
+@permission_classes((AllowAny,))
```
Comment (spasovski) [inline backend/api/views.py]: Is this safe? I imagine it's coming locally so should be fine

**Comment by spasovski** [comment]
Does this have any localhost implications for the local server?

File: backend/api/views.py (hunk 1 above)
Comment (robhudson) [reply backend/api/views.py]: The streaming API doesn't send any auth tokens so this has to be there.

**Comment by robhudson** [comment]
> Does this have any localhost implications for the local server?

I believe locally it's still using auth0 so any session that is created locally will still work.

**Pull request merged**